src/sonic-py-common/sonic_py_common/device_info.py

```diff
@@ -841,3 +844,24 @@ def is_frontend_port_present_in_host():
         if not namespace_id:
             return False
     return True
+
+
+def get_num_dpus():
+    num_dpus = 0
```

Comment:
let's add todo here. we should use platform api to get the dpu number instead of rely on the platform env config. @prgeor .

Reply:
Done